.github/main.workflow

```diff
@@ -0,0 +1,32 @@
+workflow "Build and test" {
+  on = "push"
+  resolves = ["Test-5.6", "Test-7.0", "Test-7.1", "Test-7.2" ]
+}
+
+action "Test-5.6" {
+  uses = "./.github/actions/buildtest-action"
+  env = {
+    VERSION = "5.6"
+  }
+}
+
+action "Test-7.0" {
+  uses = "./.github/actions/buildtest-action"
+  env = {
+    VERSION = "7.0"
+  }
+}
+
+action "Test-7.1" {
+  uses = "./.github/actions/buildtest-action"
+  env = {
+    VERSION = "7.1"
+  }
+}
+
+action "Test-7.2" {
+  uses = "./.github/actions/buildtest-action"
+  env = {
+    VERSION = "7.2"
+  }
+}
```

Comment:
Where is 7.3? :smile:

Reply:
Coming up, but I had to get it into master first.

Reply:
:smile: